test/test_tensor_creation_ops.py

```diff
@@ -832,6 +832,86 @@ def compare_helper_(like_fn, t):
                     tp = t.permute(p)
                     compare_helper_(like_fn, tp)
 
+    def _hvd_split_helper(self, inputs, device, dtype, dim):
+        operator_names = ["torch.vsplit", "torch.hsplit", "torch.dsplit"]
```

Comment:
This is OK. I would have passed the name, op, and reference to the helper directly instead of acquiring them from the dimension, because acquiring from the dim is a little circuitous.